custom-namespaces.yaml

```diff
@@ -1774,11 +1753,26 @@ fxci:
       type: table_view
       tables:
         - table: moz-fx-data-shared-prod.fxci_derived.task_run_costs_v1
+      measures:
+        total_cost:
+          type: sum
+          value_format_name: usd
+          sql: "${run_cost}"
```

Comment:
I'm not really sure if I should add this here or not. Within the `task_run_costs_v1` table, the `run_cost` field doesn't need to be aggregated as it already represents the total cost for a given run.

However, I'm making some explores that join this view to some of the other views. In those explores, we would want to aggregate this column across whatever dimensions we are grouping by. So I'm bit confused around whether this measure should be defined here or somewhere else (a new view in the spoke project??). Advice welcome!

Reply:
If the value is already aggregated, a SUM would be incorrect right? In that case my recommendation would be adding the columns as-is to looker-hub (this PR) and doing any joins/additional aggregations with different groupings in spoke-default.